assets/js/components/chess-board.js

```diff
@@ -13,19 +17,42 @@ class ChessBoard extends React.Component {
     store.dispatch(setGameId(gameId));
 
     $.ajax({ method: "GET", url: `/api/games/${gameId}` })
-      .then((data) => store.dispatch(setBoard(data)));
+      .then(data => {
+        store.dispatch(setPlayer(data.player));
+        store.dispatch(setGame(data));
```

Comment:
I wonder if this change would read a little clearner if you just used multiple `then` calls?

```js
.then((data) => store.dispatch(setPlayer(data.player)))
.then((data) => store.dispatch(setGame(data)))
```